chapters/syntax.tex

```diff
@@ -32,12 +32,11 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
 \textrm{S-CHAR} is any member of the Unicode character set
 (\url{http://www.unicode.org}; see \autoref{mapping-package-class-structures-to-a-hierarchical-file-system} for storing as UTF-8 on files) except double-quote """, and backslash "\textbackslash{}"
 
-For identifiers the redundant escapes (\lstinline!'\?' and '\"'!) are the same as the corresponding non-escaped
-variants (\lstinline!'?' and '"'!). {[}\emph{The single quotes are part of an
-identifier. E.g. 'x' and x are different IDENTs}{]}.
+For identifiers the redundant escapes (`\lstinline!\?!' and `\lstinline!\"!') are the same as the corresponding non-escaped
```

Comment:
I would have the single-quote inside lstinline, since the single-quotes are part of the tokens, e.g.
redundant escapes (\lstinline!'\?'! and \lstinline!'\"'!) are the same...

Reply:
My intention was slightly different, as I didn't want to write it as if the escape sequences were the entire identifier; they are typically part of a bigger identifier such as `'foo\?bar'`, and here the emphasis is on the escape sequence itself.

How about just removing the single quotes in this case?

Reply:
Ok, I can see the logic now, I'm just not sure if it is clear enough; but hopefully it is.